webapp/app/.server/db/custom-types.ts

```diff
@@ -15,6 +15,6 @@ export const bytea = customType<{
     return Buffer.from(newVal, "hex");
   },
   fromDriver(val) {
-    return val.toString("hex");
+    return `0x${val.toString("hex")}`;
```

Comment:
I found that we were recieving and returning in diffrent format, which was confusing